### ISSUE TITLE
Does evidence need to be specified in the shex?

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/go-shapes#25]
Do we need to include in the shex where evidence is added and what minimally constitutes evidence for an assertion?

**Comment by balhoff** [comment]
Yes I think so.

**Comment by cmungall** [comment]
+1

**Comment by vanaukenk** [comment]
We discussed the evidence shape on the 2019-09-18 call. 

The current shape is:

<Evidence> @<GoCamEntity> AND EXTRA a  {
  a @<EvidenceClass> {1};
  source: xsd:string {1,};
  with: xsd:string {0,1}
}// rdfs:comment  "A kind of evidence"

One question was whether source can and should ever be greater than 1.

Getting data ready for the next WB upload, I came across an error where we had two sources for a single piece of evidence.  This looks to be the result of importing an existing annotation with a pipe-separated reference (i.e. two identifiers for the same reference, one a PMID and the other the corresponding WB paper identifier).  This is a mistake, so I will remove the WB identifiers from the annotations.

Going forward, I think that just allowing one identifier per reference will help with this problem, but I wanted to note this specific case that I found.

In the meantime, I'll remove the comma after the 1 in the source entry for the evidence shape and make the pull request.
